docs/recipes/endpoint-testing-with-mongoose.md

```diff
@@ -156,3 +139,13 @@ And you're done!
 You may choose to abstract code for `test.before`, `test.beforeEach`, `test.afterEach.always` and `test.after.always` into a separate file.
 
 To see a demo of this configuration file, look at https://github.com/zellwk/ava-mdb-test
+
+## Why `test.serial` instead of `test`
```

Comment:
Why not just use a new database for each test? That's how we run tests with [agenda](https://github.com/agenda/agenda/pull/506) and we don't have any issues.